integrations/nvidia/src/haystack_integrations/components/embedders/nvidia/text_embedder.py

```diff
@@ -44,6 +47,7 @@ def __init__(
         prefix: str = "",
         suffix: str = "",
         truncate: Optional[Union[EmbeddingTruncateMode, str]] = None,
+        timeout: Optional[float] = None,
```

Comment:
This new parameter is missing in the `to_dict()`/serialization function

Reply:
Yeah good point, I was following the pattern we have for the OpenAIGenerator where we also don't save the timeout in the to_dict. But maybe that was a bug and we should do that here and there? What do you think?

Reply:
I see - I would serialize it as well in both places.

Reply:
okay sounds good!